src/widgetastic/widget.py

```diff
@@ -1194,6 +1194,10 @@ def fill(self, value):
             else:
                 self.logger.info('Filling column %r', key)
 
+            # if the row widgets aren't visible the row needs to be clicked to edit
```

Comment:
Didn't add a new constructor param, but the check against `widget.is_displayed` is much safer now, and is based off the action_row parameter and the widget attribute existing first.